homeassistant/components/openexchangerates/sensor.py

```diff
@@ -70,55 +65,59 @@ def setup_platform(
 class OpenexchangeratesSensor(SensorEntity):
     """Representation of an Open Exchange Rates sensor."""
 
-    def __init__(self, rest, name, quote):
+    _attr_attribution = ATTRIBUTION
+
+    def __init__(self, rest: OpenexchangeratesData, name: str, quote: str) -> None:
         """Initialize the sensor."""
         self.rest = rest
         self._name = name
         self._quote = quote
-        self._state = None
+        self._state: float | None = None
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return self._name
 
     @property
-    def native_value(self):
+    def native_value(self) -> float | None:
         """Return the state of the sensor."""
         return self._state
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> dict[str, Any] | None:
         """Return other attributes of the sensor."""
         attr = self.rest.data
-        attr[ATTR_ATTRIBUTION] = ATTRIBUTION
 
         return attr
```

Comment:
I think this can now be a single line:
```python
        return self.rest.data
```

Reply:
This will go away in the next PR.

Reply:
https://github.com/home-assistant/core/pull/76017